tests/periph/selftest_shield/main.c

```diff
@@ -26,6 +26,7 @@
 #include <stdlib.h>
 #include <string.h>
 
+#include "architecture.h"
```

Comment:
This one was actually needed before, but indirectly included via `#include "debug.h"`.